templates/site/node.html

```diff
@@ -40,37 +41,25 @@
                                 </tr>
                                 </tbody>
                             </table>
-
                         </div>
                     </div>
-                    <div class="col-3 pt-4">
-                        <!-- Small sharp icon next to title -->
-                        <a id="small-logo-link" target="_blank"><img id="title-icon-img"></a>
+                    <div class="col-3 col-lg-2 d-block d-lg-block">
+                        <img id="image-right" class="w-sm-100 mt-2 px-auto d-block img-fluid mw-25 mh-25 {{ img_class|default('') }}" src="{{ img_src }}" />
```

Comment:
You can remove the template strings here because you're setting the src without using a template and I don't think you're adding any additional classes dynamically here either.

Reply:
Removed the template strings